src/js/components/footer.js

```diff
@@ -3,37 +3,65 @@ const select = require("../utils/select");
 const { CLICK } = require("../events");
 const { prefix: PREFIX } = require("../config");
 
-const HIDDEN = "hidden";
 const SCOPE = `.${PREFIX}-footer--big`;
 const NAV = `${SCOPE} nav`;
 const BUTTON = `${NAV} .${PREFIX}-footer__primary-link`;
-const COLLAPSIBLE = `.${PREFIX}-footer__primary-content--collapsible`;
-
 const HIDE_MAX_WIDTH = 480;
 
+/**
+ * Expands selected footer menu panel, while collapsing others
+ */
 function showPanel() {
   if (window.innerWidth < HIDE_MAX_WIDTH) {
-    const collapseEl = this.closest(COLLAPSIBLE);
-    collapseEl.classList.toggle(HIDDEN);
-
-    // NB: this *should* always succeed because the button
-    // selector is scoped to ".{prefix}-footer-big nav"
-    const collapsibleEls = select(COLLAPSIBLE, collapseEl.closest(NAV));
+    const isOpen = this.getAttribute('aria-expanded') === 'true';
 
-    collapsibleEls.forEach((el) => {
-      if (el !== collapseEl) {
-        el.classList.add(HIDDEN);
-      }
+    // Close all other menus
+    document.querySelectorAll(BUTTON).forEach((button) => {
+      button.setAttribute('aria-expanded', false);
     });
+
+    this.setAttribute('aria-expanded', !isOpen);
   }
 }
 
+/**
+ * Swaps the <h4> element for a <button> element (and vice-versa)
+ *
+ * @param {Boolean} isMobile - If the footer is in mobile configuration
+ */
+function toggleHtmlTag(isMobile) {
+  const footer = document.querySelector(SCOPE);
+  const primaryLinks = footer?.querySelectorAll(BUTTON);
+  const newElementType = isMobile ? 'button' : 'h4';
+
+  primaryLinks?.forEach(currentElement => {
+    const currentElementClasses = currentElement.getAttribute('class');
+
+    // Create the new element
+    const newElement = document.createElement(newElementType);
+    newElement.setAttribute('class', currentElementClasses);
+    newElement.classList.toggle(`${PREFIX}-footer__primary-link--button`, isMobile);
+    newElement.textContent = currentElement.textContent;
+
+    if (isMobile) {
+      newElement.setAttribute('aria-expanded', 'false');
```

Comment:
We would need to include the `aria-controls` attribute on the button that points to the ID of each list. 

We did something similar on the File Input component where we generate a unique ID. Here's an example https://github.com/uswds/uswds/blob/7349ddfc07f3814be51f6e6143fd16b0530771f8/src/js/components/file-input.js#L109-L110

Probably best to create a new function for that and call it on `init()`.

---

We should also remove these `aria` attributes (expanded/controls) on large screens.

Reply:
> We would need to include the aria-controls attribute on the button that points to the ID of each list.

This is done. I ended up using `Math.random()` instead of `Date` because the date wasn't generating different values. 

```
 const menuId = `${PREFIX}-footer-menu-list-${Math.floor(Math.random() * 100000)}`;
```

> We should also remove these aria attributes (expanded/controls) on large screens.

Good catch on that. This is done.